livekit-agents/livekit/agents/transcription/tts_forwarder.py

```diff
@@ -92,6 +92,8 @@ def __init__(
         elif isinstance(track, (rtc.TrackPublication, rtc.Track)):
             track = track.sid
 
+        # 3.83 is the "baseline"
+        speed = speed * 3.83
```

Comment:
suggest using a different var name for this, can can you put constants at the top of the file with a comment indicating what they are?